README.md

```diff
@@ -4,7 +4,7 @@ Debugging stories are fun! This is a collection of links to various debugging st
 
 [Breakpoint takes 15ms](http://blog.jwhitham.org/2015/04/the-mystery-of-fifteen-millisecond.html)
 
-[Bug that hides from breakpoints](http://www.drewdevault.com/2014/02/02/The-worst-bugs.html)
+[Bug that hides from breakpoints](https://sircmpwn.github.io/2014/02/02/The-worst-bugs.html)
```

Comment:
It looks like http://drewdevault.com/2014/02/02/The-worst-bugs.html is the new url, the one you have in this PR is now dead as well. Would you mind fixing up and rebasing  your commit?